static/js/settings_org.js

```diff
@@ -824,6 +825,9 @@ exports.build_page = function () {
                 "#id_realm_default_twenty_four_hour_time",
             ).val();
             data.default_twenty_four_hour_time = realm_default_twenty_four_hour_time;
+        } else if (subsection === "org_profile") {
+            const realm_oss_showcase_policy = $("#id_realm_oss_showcase_policy").val();
+            data.oss_showcase_policy = realm_oss_showcase_policy;
         }
```

Comment:
I think we do not require this now as we are using integer values in option element and `populate_data_for_request` sets the correct value in data. This is used to set the correct integer from the string value of option element.